Sources/UILabelExtensions.swift

```diff
@@ -11,7 +11,20 @@
 import UIKit
 
 extension UILabel {
+
+    /// EZSwiftExtensions: fontSize: 17
+    public convenience init(x: CGFloat, y: CGFloat, w: CGFloat, h: CGFloat) {
+        self.init(x: x, y: y, w: w, h: h, fontSize: 17)
```

Comment:
Yea. The default param would resolve calls without the arg. IIRC I wrote some tests for that.